src/main/resources/ApplesAndBananas

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
```

Comment:
let's place all files for tests into src/test/resources folder

Reply:
I did this before your review, maybe I need to delete it in github?

Reply:
no, everything is ok